src/go/configgenerator/listener_generator.go

```diff
@@ -746,26 +746,25 @@ func makeTranscoderFilter(serviceInfo *sc.ServiceInfo) *hcmpb.HttpFilter {
 }
 
 func makeBackendAuthFilter(serviceInfo *sc.ServiceInfo) *hcmpb.HttpFilter {
-	var rules []*bapb.BackendAuthRule
-	for _, operation := range serviceInfo.Operations {
-		method := serviceInfo.Methods[operation]
-		if method.BackendInfo == nil || method.BackendInfo.JwtAudience == "" {
-			continue
+	var audMap map[string]bool
```

Comment:
Add comment that you are using a map to guarantee unique elements in the list.

// Determine list of unique jwt audiences.

Reply:
Done